src/back/TicketManager/src/Core/TicketManager.Core.Api/Program.cs

```diff
@@ -11,7 +12,7 @@
 using TicketManager.Core.Services.Configuration;
 using TicketManager.Core.Services.DataAccess;
 using TicketManager.Core.Services.DataAccess.Repositories;
-using TicketManager.Core.Services.JsonConverters;
+using TicketManager.Core.Services.Extensions.JsonConverters;
```

Comment:
What's with all those `Extensions` changes and why were they so crucial in this PR?

Reply:
I touched them in this PR anyway, so I also decided to decrease number of directories in main `Services` directory.